css/style.css

```diff
@@ -0,0 +1,682 @@
+  text-align: center;
+  margin-bottom: 19px;
+}
+
+.main__card-wrapper {
+  display: flex;
+  margin-bottom: 26px;
+  padding: 0px 12px;
+}
+
+.product-card {
+  position: relative;
+  min-width: 188px;
+  padding-right: 79px;
+}
+
+.product-card::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 225px;
+  bottom: 88px;
+  width: 1px;
+  height: 100px;
+  opacity: .2;
+  background: #000;
+}
+
+.product-card:last-child::after {
+  display: none;
+}
+
+.product-card__img {
+  padding: 0 10px 25px 10px;
+}
+
+.product-card__img:hover {
+  opacity: .4;
+  cursor: pointer;
+}
+
+.product-card__discr {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 11px;
+  color: #000;
+  text-align: left;
+  padding-bottom: 9px;
+}
+
+.product-card__options {
+  width: 204px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product-card__price {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 13px;
+  color: #000;
+}
+
+.product-card__sel {
+  width: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product-card__btn {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  border: none;
+}
+
+.product-card__btn:hover {
+  height: 9px;
+  width: 9px;
+  cursor: pointer;
+}
+
+.product-card__btn_selected {
+  height: 9px;
+  width: 9px;
+}
+
+.product-card__btn_grey {
+  background: #c4c4c4;
+}
+
+.product-card__btn_blue {
+  background: #41479b;
+}
+
+.product-card__btn_red {
+  background: #b62e3c;
+}
+
+.product-card__btn_yellow {
+  background: #fdb431;
+}
+
+.product-card__btn_brown {
+  background: #762e31;
+}
+
+.product-card__btn_orange {
+  background: #e29d69;
+}
+
+.product-card__btn_black {
+  background: #2c2c2c;
+}
+
+/* Медиа запросы */
+
+@media (max-width: 1024px) {
+  body {
+    min-width: 1024px;
+  }
+
+  .container {
+    max-width: 930px;
+  }
+
+  .heading {
+    margin-bottom: 24px;
+  }
+
+  .heading__logo {
+    width: 61px;
+    margin-right: 29px;
+  }
+
+  .search-form__input {
+    margin-right: 32px;
+  }
+
+  .heading__signin {
+    margin-right: 25px;
+  }
+
+  .heading__cart-wrapper {
+    margin-right: 27px;
+  }
+
+  .main__wrapper {
+    margin-bottom: 65px;
+    background: url(../img/banner-bg-1024.png);
+    height: 253px;
+  }
+
+  .main__text-wrapper {
+    padding-top: 76px;
+    padding-left: 54px;
+  }
+
+  .main__header {
+    padding-bottom: 5px;
+  }
+
+  .main__link::after {
+    left: 86px;
+  }
+
+  .main__subheader {
+    margin-bottom: 25px;
+  }
+
+.product-card {
+    padding-right: 53px;
+  }
+
+  .product-card__img {
+    height: 162px;
+    padding-bottom: 30px;
+  }
+
+  .product-card__discr {
+    margin-left: 3px;
+  }
+
+  .product-card__options {
+    margin-left: 2px;
+  }
+
+  .product-card__sel {
+    width: 55px;
+    padding-right: 14px;
+  }
+}
+
+@media (max-width: 768px) {
+  body {
+    min-width: 768px;
+    margin-top: 0px;
+  }
+
+  .heading {
+      padding: 0 4px 0 3px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5575166)